config.toml

```
@@ -143,16 +143,18 @@ theme = "elate"
       alt = "Sébastien Rémond"
 
     [[params.testimonials.item]]
-      quote = "&ldquo;Par la justesse de ses reformulations et feedbacks, Tristan a réussi à susciter au sein du groupe un changement de regard et l’envie d’oser'.&rdquo;"
-      person = 'O.L, <a href="http://franceurbaine.org" target="_blank"> France Urbaine </a> <span class="subtext">Directeur Général </span>'
+      quote = "&ldquo;Tristan a fait preuve d'une grande intelligence des situations dans un contexte de transformation majeure de l'entreprise
+      .&rdquo;"
+      person = 'Catherine Belotti, <a href="" target="_blank">
+      </a> <span class="subtext">Directrice innovation sociale Schneider Electric  </span>'
       img = "person2.jpg"
-      alt = "Verbatim"
+      alt = "Catherine Belotti"
 
     [[params.testimonials.item]]
-      quote = "&ldquo;J’avais besoin de reprendre confiance en moi dans le cadre d’une négociation. Ses grandes qualités d’écoute et d’analyse m'a aidé à mettre au point, au fil des séances de coaching, une stratégie pour gérer ma situation de manière optimale.&rdquo;"
+      quote = "&ldquo;J’avais besoin des reprendre confiance en moi dans le cadre d’une négociation. Ses grandes qualités d’écoute et d’analyse m'ont aidé à mettre au point, au fil des séances de coaching, une stratégie pour gérer ma situation de manière optimale.&rdquo;"
       person = 'Grégoire Hallé, Conservateur du Patrimoine, Musée des Beaux-Arts de Draguignan'
-      img = "person3.jpg"
-      alt = "Grégoire Hailé"
+      img = "person3.jpg"s
```

```suggestion
      img = "person3.jpg"
```
Un `s` en trop qui empêche le parsing du fichier de configuration.